docs/resources/scalr_workspace.md

```diff
@@ -42,6 +44,8 @@ The `vcs_repo` block supports:
 
 * `identifier` - (Required) A reference to your VCS repository in the format `:org/:repo`, this refers to the organization and repository in your VCS provider.
 * `branch` - (Optional) The repository branch where Terraform will be run from. Default `master`.
+* `path` - (Optional) The repository sub-directory that Terraform will execute from. If omitted or submitted as an empty string, this defaults to the repository's root.
```

Comment:
@rtfee only we need to update our public documentation as new attributes are available now